mbed_greentea/mbed_greentea_cli.py

```diff
@@ -228,6 +230,10 @@ def main():
                     dest='hooks_json',
                     help='Load hooks used drive extra functionality')
 
+    parser.add_option('', '--test-spec',
+                    dest='tests_spec',
```

Comment:
Slight inconsistency in the naming here:
--test-spec
tests_spec 

Might be better to make them the same ?